Apps/W1/HybridSL/app/src/pages/MigrationErrorOverview.Page.al

```diff
@@ -0,0 +1,164 @@
+                }
+                field(LastRecordUnderProcessing; Rec."Last Record Under Processing")
+                {
+                    ApplicationArea = All;
+                    Caption = 'Last Processed Record';
+                    Editable = false;
+                    ToolTip = 'Specifies the last record that was processed before the error occurred.';
+
+                    trigger OnDrillDown()
+                    begin
+                        Message(Rec.GetLastRecordsUnderProcessingLog());
+                    end;
+                }
+                field(StackTrace; StackTraceTxt)
+                {
+                    ApplicationArea = All;
+                    Caption = 'Error Stack Trace';
+                    ToolTip = 'Specifies the stack trace that relates to the error.';
+                    trigger OnDrillDown()
+                    var
+                        MessageWithStackTrace: Text;
+                        NewLine: Text;
+                    begin
+                        NewLine[1] := 10;
+                        MessageWithStackTrace := StackTraceTxt + NewLine + Rec.GetExceptionCallStack();
+                        Message(MessageWithStackTrace);
+                    end;
+                }
+                field(ErrorDismissed; Rec."Error Dismissed")
+                {
+                    ApplicationArea = All;
+                    ToolTip = 'Specifies whether the error has been dismissed.';
+                }
+            }
+        }
+    }
+    actions
+    {
+        area(Processing)
+        {
+            action(OpenInCompany)
+            {
+                ApplicationArea = All;
+                Caption = 'Open in Company';
+                Image = Open;
+                Promoted = true;
+                PromotedCategory = Process;
+                PromotedOnly = true;
+                ToolTip = 'Open the company in which the error occurred.';
+                Visible = false;
+
+                trigger OnAction()
+                begin
+                    Hyperlink(GetUrl(ClientType::Web, Rec."Company Name", ObjectType::Page, Page::"Data Migration Overview"));
+                end;
+            }
+            action(ShowProcessedRecordsLog)
+            {
+                ApplicationArea = All;
+                Caption = 'Show log of processed records';
+                Image = ShowList;
+                Promoted = true;
+                PromotedCategory = Process;
+                PromotedOnly = true;
+                ToolTip = 'Shows the log of last processed records before error occured.';
+
+                trigger OnAction()
+                begin
+                    Message(Rec.GetLastRecordsUnderProcessingLog());
+                end;
+            }
+            action(DismissError)
+            {
+                ApplicationArea = All;
+                Caption = 'Dismiss Error';
+                Image = CompleteLine;
+                Promoted = true;
```

Comment:
Please update all pages to new actionref syntax. Search for Promoted = true and invoke the code action to convert.

Reply:
All pages have been updated to new actionref syntax